### PR TITLE
Add Quiet Build Flag

### LINKED ISSUE #6028: docker-compose: --quiet for builds
```sh
$ docker -v
Docker version 18.05.0-ce, build f150324
$ docker-compose -v
docker-compose version 1.21.2, build a133471
```

I want to silence the output of `docker-compose build` just like there is a `--quiet` command in `docker` if so, please display the full command required to get quiet builds.

I guess one could just redirect standard out, but I'd rather not do this.

Many thanks,

### PR DESCRIPTION
**Pull request by akshitgrover**
Resolves #6028 
It adds the quiet build flag for the docker-compose build command,
Basically, It silents all the data in STDOUT, not STDERR.
Therefore, Any successful build will not print anything to the STDOUT, But any error will be consoled out.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -49,7 +49,7 @@ directory = coverage-html
 [flake8]
 max-line-length = 105
 # Set this high for now
-max-complexity = 11
+max-complexity = 14
```
Comment (shin-) [inline tox.ini]: Please do not modify this setting. If a method's complexity is reported as being too high after your changes, break it up.

File: tests/fixtures/build-quiet-cli/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FROM busybox:latest
```
Comment (shin-) [inline tests/fixtures/build-quiet-cli/Dockerfile]: I don't think a new fixture is necessary here - just reuse `build-args` or `build-ctx`

File: compose/service.py [hunk 3]
```diff
@@ -1074,7 +1076,12 @@ def build(self, no_cache=False, pull=False, force_rm=False, memory=None, build_a
         )
 
         try:
-            all_events = stream_output(build_output, sys.stdout)
+            if not quiet:
+                all_events = stream_output(build_output, sys.stdout)
+            else:
+                all_events = []
```
Comment (shin-) [inline compose/service.py]: You can simplify this by using the same trick as in `service.pull`: https://github.com/docker/compose/blob/f207d94b3c1be9bf83bf5421d1c88043eeedc650/compose/service.py#L1190-L1196

File: compose/cli/main.py [hunk 4]
```diff
@@ -285,6 +286,7 @@ def build(self, options):
             build_args=build_args,
             gzip=options.get('--compress', False),
             parallel_build=options.get('--parallel', False),
+            quiet=options.get('--quiet', False)
```
Comment (shin-) [inline compose/cli/main.py]: please use `silent` as the parameter name internally to match what we did with `pull`

**Comment by akshitgrover** [comment]
Hey @shin- , Thank you for the review, Will update the pull request with proposed changes.

**Comment by shin-** [comment]
Thanks! It's a good indicator that we probably need to refactor the method, but you don't need to do anything about it for now.

Can you rebase your PR against the most recent version of `master` to get rid of the merge commit, and squash the remaining commits into one? After that we should be good to go.

**Comment by akshitgrover** [comment]
Hey @shin- 
What should I do, How to proceed further?

**Comment by iBoB-Code** [comment]
Hey guys ! Any updates on this pull request being accepted one day? 
Cheers!

**Comment by akshitgrover** [comment]
@chris-crone what should be the changes for this to get merged?

**Comment by chris-crone** [comment]
Thanks for the PR @akshitgrover and sorry that we've been slow to react on this one. It LGTM but to get it merged we'll need to get it green on the CI.

Can you please rebase on the master branch?

**Comment by chris-crone** [comment]
One thing that is missing is bash completion for this flag.

**Comment by akshitgrover** [comment]
@chris-crone I'll surely make the desired changes.
Thank you :)

**Pull request closed without merging**